packages/v4/src/content/design-guidelines/components/time picker/time-picker.md

```diff
@@ -0,0 +1,104 @@
+---
+id: Time picker
+section: components
+related: ["Calendar month", "Date picker"]
+---
+
+A **time picker** allows users to select a time from a list of options. Time can be displayed in hours and minutes, but not in seconds.
+
+## Elements
+
+<img src="./img/Time picker- elements.png" alt="Time picker elements" width="362"/>
+
+1. **Time field**: Displays a user’s selected time. Always fill the time field with a placeholder time format (HH:MM) or a pre-selected time before users select their time by entering the field, typing ahead, scrolling, and selecting their desired time from the menu. The time field should never be empty.
+2. **Time field in focused state**: Opens a menu list of time options. 
+3. **Menu**: Contains a list of times in set increments, such as 12:00 AM to 11:30 PM in 30 minute increments.
+4. **Menu item in hovered state**
+5. **Menu item in disabled state**
+
+## Usage
+Use time pickers to enable users to select or input a time for use cases like item filtration or scheduling in a form.
```

Comment:
Re-word:

"Use time pickers to enable users to select or input a time. For example, in a toolbar for item filtration, or in a form for scheduling inputs."